domain-cc/cc-app/src/python_src/api.py

```diff
@@ -88,3 +91,75 @@ def get_classification(claim: Claim) -> Optional[PredictedClassification]:
 
     logging.info(f"classification: {classification}")
     return classification
+
+
+def generate_flattened_claim(contention_text, diagnostic_code, multi_contention_claim):
+    print(f"got dc: {diagnostic_code}")
+    return FlattenedSingleIssueClaim(
+        claim_id=multi_contention_claim.claim_id,
+        form526_submission_id=multi_contention_claim.form526_submission_id,
+        diagnostic_code=diagnostic_code,
+        claim_type=multi_contention_claim.claim_type,
+        contention_text=contention_text,
+    )
+
+
+@app.post("/classifier", deprecated=True)
+def classifier_v1(
+    claim: FlattenedSingleIssueClaim,
+) -> Optional[PredictedClassification]:
+    logging.info(
+        f"claim_id: {claim.claim_id}, form526_submission_id: {claim.form526_submission_id}"
+    )
+    return do_get_classification(claim)
+
+
+@app.post("/classifier/v2")
```

Comment:
We may want to create a separate `api.py` module for `v2`, and/or have the version number before the controller name here, especially if we anticipate interdependencies between routes.

Reply:
alternatively, would moving 
```
@app.post("/classifier", deprecated=True)
def classifier_v1(
```
to a deprecated folder achieve the same purpose?